content/blog/2016/2016-08-26-ask-the-experts-jenkins-world.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Ask the Experts at Jenkins World 2016"
+tags:
+- jenkinsworld
+- event
+author: rtyler
+---
+
+
+image:/images/conferences/Jenkins-World_125x125.png[role=right]
+
+Our events officer link:https://github.com/alyssat[Alyssa] has been working for
+the past number of weeks to organize the "Open Source Hub" at
+link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World 2016]. The Hub
+is a location on the expo floor where members of the Jenkins project can hang
+out, share demos and help Jenkins users via the "Ask the Experts" program. Thus
+far, we have a great list of experts who have volunteered to help staff the
+booth which includes many frequent contributors, link:/projects/jam[JAM]
+organizers and board members.
+
+A few of the friendly folks you will see at Jenkins World are:
+
+* *link:https://github.com/p4paul[Paul Allen]* -
+  link:https://wiki.jenkins-ci.org/display/JENKINS/P4+Plugin[P4 Plugin]
+  maintainer and link:/doc/pipeline[Pipeline] contributor.
+* *link:https://github.com/rtyler[R Tyler Croy]* -
+  link:https://github.com/jenkins-infra[Jenkins infrastructure] maintainer and
+  board member.
+* *link:https://github.com/jglick[Jesse Glick]* - link:/doc/pipeline[Pipeline]
+  maintainer and long-time contributor to Jenkins
+  link:https://github.com/jenkinsci/jenkins[core]
+* *link:https://github.com/eddumelendez[Eddú Meléndez Gonzales]* - Organizer for
+  the link:http://www.meetup.com/es/Lima-Jenkins-Area-Meetup/[Lima (Perú)
+  Jenkins Area Meetup] and contributor to link:http://spring.io[Spring].
+* *link:https://github.com/jh86[Jon Hermansen]* - Organizer for the
+  link:http://www.meetup.com/Los-Angeles-Jenkins-Area-Meetup/[Los Angeles
+  Jenkins Area Meetup] developer and Pipeline user.
```

Comment:
"Los Angeles Jenkins Area Meetup, developer [optional Oxford comma] and Pipeline user."